desktop/packages/windows/wix/ruffle.wxs

```diff
@@ -0,0 +1,182 @@
+                    </CreateFolder>
+                </Component>
+
+                <Component Id='License' Guid='*'>
+                    <File Id='LicenseFile'
+                          DiskId='1'
+                          Source='../../../../LICENSE.md'
+                          KeyPath='yes'/>
+                </Component>
+
+                <Directory Id='Bin' Name='bin'>
+                    <Component Id='Path' Guid='ABEDD436-5D52-4C25-9FD1-D810A2B66AC1' KeyPath='yes'>
+                        <Environment
+                                Id='PATH'
+                                Name='PATH'
+                                Value='[Bin]'
+                                Permanent='no'
+                                Part='last'
+                                Action='set'
+                                System='yes'/>
+                    </Component>
+                    <Component Id='binary0' Guid='*'>
+                        <File
+                                Id='DesktopEXE'
+                                Name='ruffle.exe'
+                                DiskId='1'
+                                Source='$(CargoBuildDir)/ruffle_desktop.exe'
+                                KeyPath='yes'>
+                            <Shortcut Name="Ruffle" Directory="ProgramMenuFolder" Advertise="yes"
+                                      WorkingDirectory="INSTALLFOLDER" Icon="Icon.ico"/>
+                            <Shortcut Name="Ruffle" Directory="DesktopFolder" Advertise="yes"
+                                      WorkingDirectory="INSTALLFOLDER" Icon="Icon.ico"/>
+                        </File>
+                    </Component>
+                </Directory>
+            </Directory>
+        </StandardDirectory>
+
+        <Component Id='Associations' Guid='2216512a-9c76-412f-ba36-08e703a56f25'>
+            <!-- 
+                This creates the Ruffle progids and *hints* that Ruffle can be associated with them.
+                This may or may not cause Windows to make the association, but more likely makes it ask the user 
+                what they want the next time they open a swf.
+                -->
+
+            <ProgId Id='Ruffle.swf' Description='Flash Movie' Icon="DesktopEXE" IconIndex="1">
+                <Extension Id='swf' ContentType='application/x-shockwave-flash'>
+                    <Verb Id='open' Command='Open' TargetFile='DesktopEXE' Argument='"%1"'/>
+                </Extension>
+            </ProgId>
+            <ProgId Id='Ruffle.spl' Description='Flash Movie' Icon="DesktopEXE" IconIndex="1">
+                <Extension Id='spl' ContentType='application/x-shockwave-flash'>
+                    <Verb Id='open' Command='Open' TargetFile='DesktopEXE' Argument='"%1"'/>
+                </Extension>
+            </ProgId>
+            <ProgId Id='Ruffle.ruf' Description='Ruffle Bundle' Icon="DesktopEXE" IconIndex="1">
+                <Extension Id='ruf' ContentType='application/x-ruffle-bundle'>
```

Comment:
I proposed one day `application/x.ruffle-bundle+zip`: according to my knowledge `x-` is obsolete, and `+zip` signifies that the format is based on a zip file.

Reply:
I tried to find your proposal but discord is awful to search, couldn't find it! I'll update to your suggestion

(I don't think this actually affects anything though, I have no idea why it's needed...)